syz-fuzzer/fuzzer.go

```diff
@@ -261,6 +258,18 @@ func main() {
 	fuzzer.pollLoop()
 }
 
+func (fuzzer *Fuzzer) useBugFrames(r *rpctype.ConnectRes, flagProcs int) {
+	var gateCallback func()
+	if r.CheckResult.Features[host.FeatureLeakChecking].Enabled {
+		gateCallback = func() { fuzzer.gateCallback(r.MemoryLeakFrames) }
+	}
+	fuzzer.gate = ipc.NewGate(2*flagProcs, gateCallback)
+
+	if r.CheckResult.Features[host.FeatureKCSAN].Enabled {
```

Comment:
Does it make sense to check `len(r.DataRaceFrames) != 0`?

Reply:
Done.